server/context/conf.js

```diff
@@ -40,6 +40,11 @@ if (IS_COVERAGE_ACTIVE) {
     configuration = JSON.parse(configurationString);
     Log.info('[Configuration] ', configuration);
   } catch (e) {
+    if (e instanceof SyntaxError) {
+      let errMsg = `Error: ${coverageFile} is not a valid JSON`;
+      console.error(errMsg, e);
+      Log.error(errMsg, e, e.stack);
```

Comment:
I would replace this line into something more relevant, because console.error will always be printed. What do you think about just `Log.error(e.stack);` ?